### PR TITLE
apply crud cart functionality with amazon

### PR DESCRIPTION
**Pull request by peerwit**

### CODE REVIEW

**Comment by scottrice10** [comment]
This is currently unmergeable:
1. server.js file is wrong. Please run grunt serve before making pull requests.
2. Delete all commented code now. This makes the code very confusing to work with, even at this stage in the development process.

File: server.js [hunk 1]
```diff
@@ -57,17 +57,17 @@ app.all('*', function(req, res, next) {
...
 //   next();
 // });
 
-// serverHTTP.listen(config.port, config.ip, function () {
+// serverHTTP.listen(9000, config.ip, function () {
 //     console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
 // });
 
-var socketio = require('socket.io')(serverHTTPS, {
+var socketio = require('socket.io')(serverHTTP, {
```
Comment (scottrice10) [inline server.js]: serverHTTPS!!!!

**Comment by kxprim** [comment]
:+1:

**Comment by scottrice10** [comment]
:+1:

**Pull request merged**